multires_image/CMakeLists.txt

```diff
@@ -79,9 +79,9 @@ ament_target_dependencies(${PROJECT_NAME}
   gps_msgs
   mapviz
   pluginlib
-  rclcpp 
+  rclcpp
```

Comment:
Small change, but as along as we're dropping the extraneous line endings, could we get these dependencies in the right alphabetical order?

Reply:
The dependencies should now be listed in the correct order.